docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,104 @@
+1. **Simplicity**: Reduce scope of dashboards and resources available.
+1. **Cost attribution**: Track and bill costs to their customers, departments, or divisions.
+1. **Customization**: Each team could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana Teams
+
+It makes sense to organize users in Teams in order to assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team. A user can belong to multiple teams.
+
+A user can be a Member or an Administrator for a given team. Members of a team inherit permissions from the team, but they cannot edit the team itself. Team Administrators can add members to a team and update its settings, such as the team name, team member’s team roles, UI preferences, and the default dashboard to be displayed upon login for the team members.
+
+Teams is a perfect solution to work with a small subset of users. Teams can share resources among other teams.
+
+### Users in Grafana Organizations
+
+[Grafana Organizations] was born as a concept to isolate users from dashboards and datasources by having multiple organizations under a single instance of Grafana. This means that users under different organizations won't share any resources such as dashboards, folders, and datasources.
+
+Grafana Organizations provide a mesure of isolation within Grafana by default. The intention is to present different user experiences, which give the sense of different instances of Grafana within a single instance. However, **we recommend aginast Grafana Organizations** because they lack the scalability of [Folders].
+
+### Choosing between teams and organizations
+
+Grafana Teams and Grafana Organizations serve similar purposes in the Grafana platform. Both are designed to help manage and control access to resources, Teams provide a more focused approach for smaller groups or projects. Teams allow for easy collaboration and shared ownership of resources among members, making it simple to manage access and control on who has access to said resources.
+
+Organizations, on the other hand, provide a higher level of management for multiple teams and resources, and are designed for larger enterprises or organizations with a complex setup or multiple and different business models.
+
+That being said, Teams provide many of the same benefits as Organizations, and can be seen as a more streamlined and simplified approach to managing resources and access. As such, it is possible that in the future, Teams may eventually replace Organizations as the primary way to manage resources in Grafana.
+
+## Do I have external systems?
+
+Consider the need for [M2M] communications. If there's a system that needs to interact with Grafana, ensure that it has proper access for it.
```

Comment:
```suggestion
Consider the need for machine-to-machine (M2M) communications. If a system needs to interact with Grafana, ensure it has proper access.
```

Reply:
Thank you for the suggestion! I'm updating the text with it. However, I'm keeping the [M2M] as a placeholder for the terminology section.